http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -401,7 +402,7 @@ final case class RuntimePatchDetails(runtimeId: Long, runtimeStatus: RuntimeStat
 
 final case class PubsubException(message: String) extends WorkbenchException(message)
 
-object LeoPubsubCodec {
+object LeoPubsubCodec extends AzureObjectCodec {
```

Comment:
Added an "AzureObjectCodec" for encoding LandingZoneResources in the pubsub message, and extended it hear to make the encoders and decoders implicitly visible